webapp/security/views.py

```diff
@@ -186,24 +187,13 @@ def feed_entry(notice, url_root):
 
 
 @authorization_required
-def api_create_notice():
+def create_notice():
     if not flask.request.json:
         return (flask.jsonify({"message": f"No payload received"}), 400)
 
-    # Because we get a dict with ID as a key and the payload as a value
-    notice_id, payload = flask.request.json.popitem()
-
-    notice = db_session.query(Notice).filter(Notice.id == notice_id).first()
-    if notice:
-        return (
-            flask.jsonify({"message": f"Notice '{notice.id}' already exists"}),
-            400,
-        )
-
-    notice_schema = NoticeSchema()
-
     try:
-        data = notice_schema.load(payload, unknown=EXCLUDE)
+        notice_schema = NoticeSchema()
```

Comment:
Why move this inside the `try`? Is this line likely to lead to a `ValidationError`?